framework/tests/h1h2checker/CompoundStringAssignment.java

```diff
@@ -9,7 +9,6 @@ void test1() {
         String local = null;
         // There was a bug in data flow where
         // the type of local was @H1Bot @H2Bot after this
-        // StringConcatenateAssignmentNode,
```

Comment:
The sentence doesn't make sense if you remove the class name. What is this saying?

Reply:
fixed.